src/schema/settings.ts

```diff
@@ -322,7 +357,10 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         const settings = await repo.findOneBy({ userId });
 
         if (!settings) {
-          return repo.save({ ...data, userId });
+          return repo.save({
+            ...data,
+            userId,
```

Comment:
I think we will need `updateFlagsStatement` because if request does not send everything the whole `flags` field will get overwritten.

Reply:
Let me change this and add failing test first, good catch 🥳